src/components/Card/Card.jsx

```diff
@@ -10,46 +9,44 @@ function Card({
   domain,
   articleTitle,
   readingTime,
+  url,
   isDeleteMode,
+  deleteArticle,
 }) {
-  const articleFavicon = favicon || readimFavicon;
+  const animation = isDeleteMode ? "" : "hover:scale-115";
 
   return (
-    <li className="relative content-center w-40 h-40 pl-5 list-none transition-all delay-75 bg-white shadow-md group hover:scale-125 shadow-black/25 rounded-3xl">
-      <a
-        href="https://www.google.com"
-        target="_blank"
-        className="relative -top-1"
-        rel="noreferrer"
-      >
-        <div>
-          <img
-            className="inline-block w-4"
-            src={articleFavicon}
-            alt="favicon"
-          />
-          <p className="text-[11px] inline-block ml-1 font-medium">{domain}</p>
+    <li
+      className={`relative flex flex-col content-center w-48 p-5 list-none transition-all bg-white shadow-md h-50 shadow-black/25 rounded-3xl group ${animation}`}
+    >
+      <a href={url} target="_blank" className="relative" rel="noreferrer">
+        <div className="flex">
+          <img className="inline-block w-4" src={favicon} alt="favicon" />
+          <p className="text-[11px] w-4/5 truncate inline-block ml-1 font-medium">
+            {domain}
+          </p>
         </div>
         <ReadingTime readingTime={readingTime} />
-        <p className="mt-1 mr-5 group-hover:pointed-title font-extralight title-ellipsis">
+        <p className="mt-6 mr-1 text-sm leading-sung group-hover:pointed-title font-extralight title-ellipsis">
```

Comment:
```suggestion
        <p className="mt-6 mr-1 text-sm leading-snug group-hover:pointed-title font-extralight title-ellipsis">
```
tailwind css인 `leading-snug` 으로 변경해주시거나 삭제하셔도 될 것 같습니다!